data/json/recipes/recipe_ammo.json

```diff
@@ -575,6 +575,78 @@
       [ "cash_card", 2 ]
     ]
   ]
+},{
+  "type" : "recipe",
+  "result" : "arrow_metal_bodkin",
+  "category" : "CC_AMMO",
+  "subcategory": "CSC_AMMO_ARROWS",
+  "skill_used" : "fabrication",
+  "skills_required": [[ "archery", 5 ], [ "survival", 1 ]],
+  "difficulty" : 5,
+  "time" : 20000,
+  "autolearn" : true,
+  "book_learn" : [[ "recipe_arrows", 4 ]],
+  "qualities" : [
+    {"id":"CUT","level":1},
+    {"id":"HAMMER","level":1},
+    {"id":"SAW_M","level":1}
+  ], "components" : [
+    [
+      [ "adhesive", 1, "LIST" ],
+      [ "filament", 20, "LIST" ]
+    ],
+    [
+      [ "scrap", 9 ],
+      [ "steel_chunk", 3 ]
+    ],
+    [
+      [ "feather", 10 ],
```

Comment:
I suggest creating a new `fletching` requirement list and replacing it in all arrows recipes.

Reply:
That's not a bad idea. I'll look into it.

Reply:
There, I put it in.